acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/signer/runner/Web3SignerRunner.java

```diff
@@ -183,18 +186,23 @@ public void shutdown() {
     params.add(Boolean.toString(signerConfig.isSlashingProtectionEnabled()));
 
     if (signerConfig.isSlashingProtectionEnabled()) {
-      try {
-        slashingDatabase = EmbeddedPostgres.start();
-        createSlashingDatabase(slashingDatabase.getPostgresDatabase());
-      } catch (final IOException e) {
-        throw new RuntimeException("Unable to start embedded postgres db", e);
+      final EmbeddedPostgres slashingDatabase;
+      if (signerConfig.getSlashingProtectionDbUrl() == null) {
+        try {
+          slashingDatabase = EmbeddedPostgres.start();
+          createSlashingDatabase(slashingDatabase.getPostgresDatabase());
+        } catch (final IOException e) {
+          throw new RuntimeException("Unable to start embedded postgres db", e);
+        }
+        slashingProtectionDbUrl =
+            String.format("jdbc:postgresql://localhost:%s/postgres", slashingDatabase.getPort());
```

Comment:
nit: can this be extracted into a function say createEmbeddedDatabase

Reply:
Done